src/Modules/localStorage.js

```diff
@@ -0,0 +1,10 @@
+export const addLocalStorage = (arrBooks) => {
+  const storage = localStorage.setItem('book', JSON.stringify(arrBooks));
+  return storage;
+};
+export const getLocalStorage = () => {
+  const storage = localStorage.getItem('book') === null
+    ? []
+    : JSON.parse(localStorage.getItem('book'));
+  return storage;
+};
```

Comment:
🍰 Good work on implementing clean code to store data in the local storage, this is impressive